fn foo(): Int {

    return 1
    print("Unreachable!")
}

fn main() {
    foo()
}